<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="infoTab">
        <attribute name="title">
         <string>Video Info</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="infoTable">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>100</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>File</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Frame</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Time(ms)</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Progress</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Width</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Height</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Frame Rate</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Total Frames</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Property</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>File</string>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>Frame</string>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>Time(ms)</string>
            </property>
           </item>
           <item row="2" column="1">
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>Progress</string>
            </property>
           </item>
           <item row="3" column="1">
            <property name="text">
             <string>0.00%</string>
            </property>
           </item>
           <item row="4" column="0">
            <property name="text">
             <string>Width</string>
            </property>
           </item>
           <item row="4" column="1">
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item row="5" column="0">
            <property name="text">
             <string>Height</string>
            </property>
           </item>
           <item row="5" column="1">
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item row="6" column="0">
            <property name="text">
             <string>Frame Rate</string>
            </property>
           </item>
           <item row="6" column="1">
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item row="7" column="0">
            <property name="text">
             <string>Total Frames</string>
            </property>
           </item>
           <item row="7" column="1">
            <property name="text">
             <string>0</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="video">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
          <property name="text">
           <string>No video selected</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="playControls">
          <item>
           <widget class="QPushButton" name="playButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>⏵</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="playSlider">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="progressText">
            <property name="text">
             <string>00:00/∞</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="openStream"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="showSidebar"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="openFile">
   <property name="text">
    <string>&amp;Open Video...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="openStream">
   <property name="text">
    <string>Open &amp;RTSP</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="showSidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Sidebar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
